git-server/src/lib.rs

```diff
@@ -385,3 +393,41 @@ fn gen_random_string() -> String {
         .map(char::from)
         .collect()
 }
+
+/// Grab projects names and populate map<name, urn> with them
+fn grab_aliases(options: &Options , map: &mut HashMap<String, String>) {
+    use librad::paths::Paths;
+    use librad::git::identities;
+    use librad::git::storage::read::ReadOnly;
+    use librad::git::identities::SomeIdentity::Project;
+    use radicle_source::surf::vcs::git::namespace::Namespace;
+
+    // NOTE: is it always safe to call unwrap on options.root?
+    let paths = Paths::from_root(options.root.as_ref().unwrap()).unwrap();
+    let storage = ReadOnly::open(&paths).expect("failed to read storage paths");
+
+    let mut duplicates: Vec<String> = Vec::new();
+    let identities = identities::any::list(&storage)
+        .expect("Failed to list identities in storage");
+    for identity in identities {
+        match identity {
+            Ok(Project(project)) => {
+                let urn = Namespace::try_from(project.urn().encode_id().as_str())
+                    .unwrap().to_string();
+                let name = (&project.payload().subject.name).to_string() + ".git";
+                tracing::info!("alias \"{}\" for \"{}\"", name, urn);
```

Comment:
If you use `"{:?}"` on a string, it will be quoted.

Reply:
Fixed https://github.com/radicle-dev/radicle-client-services/pull/74/commits/8e795c8bba5f1e1edb21d3dd531c6a8bea3d75ff#diff-b672a15ad14f54528cebd77aee76d3007962d44bda987b3e0a34597ec80fd6b1R134